packages/react/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -149,8 +149,16 @@ export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyle
     daySelected: [
       dateRangeType !== DateRangeType.Month && {
         backgroundColor: palette.neutralLight + '!important',
-        border: `1px solid ${palette.neutralSecondary}`,
         selectors: {
+          ['&:after']: {
```

Comment:
Nit, you don't need the square brackets around the string (here or elsewhere)
```suggestion
          '&:after': {
```